### PR TITLE
Modifiable animation delay slide

### PR DESCRIPTION
**Pull request by tripwirebe**
Implemented a slider which will modify the animation delay. this makes the animation smoother or harsher.

### CODE REVIEW

**Comment by ZZZZzzzzac** [comment]
sry for late response, I haven't taken care of this repo for quite long time. thanks for contribution anyway :)

**Comment by tripwirebe** [comment]
Glad to be of help. Looking forward to see these updates flow through to the steam workshop.
I have a few more ideas for some other features. Would you be open to accept those if I would implement them on my fork?

**Comment by ZZZZzzzzac** [comment]
Any contribution is welcomed! 
But I know little about JS and web developement so I'm may not be able to review code. I will test it and push to workshop when I have time

File: js/index.js [hunk 1]
```diff
@@ -10,14 +10,26 @@ var d = void 0,
...
+
+function padClock(previous, _current) {
+	return previous + ('0' + _current).slice(-2);
 }
 
 function getClock() {
-	d = new Date();
-	return [use24HourClock ? d.getHours() : d.getHours() % 12 || 12, d.getMinutes(), d.getSeconds()].reduce(padClock, '');
+	date = new Date();
+	return [use24HourClock ? date.getHours() : ddate.getHours() % 12 || 12, date.getMinutes(), date.getSeconds()].reduce(padClock, '');
```
Comment (tripwirebe) [inline js/index.js:32]: oops there is a bug in the twelve hour system. I used ddate instead of date.

**Pull request merged**